byron-proxy/src/exec/Validator.hs

```diff
@@ -139,14 +139,16 @@ main = do
           Just fp -> fp
         rnm = requiresNetworkMagic opts
     -- Copied from validate-mainnet in cardano-ledger.
-    let (Right genHash) = decodeAbstractHash
+    let genHash = either (error . show) id $
```

Comment:
If that code ever fails, we would like some more information about where the `error` call originated. At the very least, include the function name or better yet use `Mondule.function`.

Reply:
@erikd, I think that's already covered -- that's how the error looks like:
```
validator: GenesisHashMismatch (GenesisHash {unGenesisHash = AbstractHash c6a004d3d178f600cd8caa10abbebe1549bef878f0665aea2903472d5abf7323}) (AbstractHash 89d9b5a5b8ddc8d7e5a6795e9774d97faf1efea59b2caf7eaf9f8c5b32059df4)
CallStack (from HasCallStack):
  error, called at src/exec/Validator.hs:148:30 in main:Main
```

I actually used this for debugging.  If you think that still can be improved, do tell!